README.md

```diff
@@ -90,4 +88,6 @@ cd src/data
 pip install --no-cache-dir -r requirements.txt
 python3 scripts/process-data.py
 python3 scripts/add-context-by-property-type.py
+python3 scripts/clean-and-pare-down-data_all_years.py
```

Comment:
@SuragNuthulapaty - are you sure this order is correct? Also the new files are in the source directory, not the dist directory, which means they aren't actually being used for the main site itself, just `building-benchmarks.csv`. Is that correct?